Assets/Scripts/PowerSliderBehaviour.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PowerSliderBehaviour : MonoBehaviour {
+
+	public Slider slider;
+	public float sliderSpeed = .05f;
+	private bool pressed;
```

Comment:
Can we make this `isPressed` or `isFlingPressed`?

Reply:
Also, should we initialize the bool here, or is that not possible since it's private?

Reply:
I think private variables should be initialized in start but that might just my style